src/irr/asset/CGLILoader.cpp

```diff
@@ -150,6 +150,8 @@ namespace irr
 					region->imageExtent.width = texture.extent(regionIndex).x;
 					region->imageExtent.height = texture.extent(regionIndex).y;
 					region->imageExtent.depth = texture.extent(regionIndex).z;
+					region->bufferRowLength = IImageAssetHandlerBase::calcPitchInBlocks(region->imageExtent.width, texelBlockByteSize);
```

Comment:
in texels not blocks :(

Reply:
the input is in texels as far as I know

Reply:
so computed value is in texels as well

Reply:
this is  a GLI loader!!!!! It loads BC1-7 images!

Reply:
there was a mistake, fixed